<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SN</class>
 <widget class="QGroupBox" name="SN">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>123</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(00,00,00);
color: #FFFFFF;
}
QPushButton{
	background-color: black;
}
 QPushButton:pressed {
     background-color: rgb(224, 0, 0);     
}
QPushButton:hover {
     background-color: rgb(100, 30, 30);
}
QPushButton#startButton:disabled:checked {
	color: gray;
}
QPushButton#startButton:disabled:checked {
	background-color: #ffaaaa;
}
QPushButton#startButton:disabled:!checked {
	background-color: lightGreen;
}</string>
  </property>
  <widget class="QLineEdit" name="globalY">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    color: yellow;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="globalZ">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    color: yellow;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="globalX">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    color: yellow;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="localX">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="convert">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>40</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>â†”</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="localZ">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>40</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="localY">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>40</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="ClearBtn">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>50</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>globalX</tabstop>
  <tabstop>globalY</tabstop>
  <tabstop>globalZ</tabstop>
  <tabstop>convert</tabstop>
  <tabstop>localX</tabstop>
  <tabstop>localY</tabstop>
  <tabstop>localZ</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
